### PR TITLE
Fixing flag user modal

### PR DESCRIPTION
**Pull request by Rafi993**
## What type of PR is this?

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
"Flag user" button was working only first time in /mod route. This PR fixes the way event listener is attached to that button similar to "Unpublish Article" button fixing this issue.

## Related Tickets & Documents

Closes https://github.com/forem/forem/issues/10605

## QA Instructions, Screenshots, Recordings
1. Go to /mod
2. Open article flag user
3. Open another article and flag another user 
4. Go to home feed
5. Click on an article
6. Flag the author of that article

## Added tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [x] I need help with writing tests

## Added to documentation?

- [ ] Docs.forem.com
- [ ] README
- [x] No documentation needed

### CODE REVIEW

File: app/javascript/actionsPanel/actionsPanel.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { toggleFlagUserModal } from '../packs/flagUserModal';
 import { request } from '@utilities/http';
```
Comment (Rafi993) [inline app/javascript/actionsPanel/actionsPanel.js:2]: code climate is complaining about this order of imports but when I change the order and commit the file eslint in husky hook changes this order back.

**Comment by Rafi993** [comment]
I think some unrelated test is failing https://travis-ci.com/github/forem/forem/jobs/423501292#L1420

**Comment by ludwiczakpawel** [comment]
@Rafi993 I've restarted the build :)

**Review by ludwiczakpawel — APPROVED** [review]
It does work so 👍 from me. I'll trust someone else to check the code :)

**Review by benhalpern — APPROVED** [review]
Looks good to me. I invalidated the codeclimate complaint.

**Pull request merged**